src/base-faker.ts

```diff
@@ -0,0 +1,215 @@
+import type { Mersenne } from './internal/mersenne/mersenne';
+import mersenne from './internal/mersenne/mersenne';
+import { DatatypeModule } from './modules/datatype';
+import { BaseDateModule } from './modules/date';
+import { BaseHelpersModule } from './modules/helpers';
+import { NumberModule } from './modules/number';
+import { StringModule } from './modules/string';
+
+/**
+ * This is Faker's base class containing only modules that can be used to generate basic data.
```

Comment:
Maybe change this to something like:

```suggestion
 * This is Faker class that doesn't need any localized data to generate it's output. It should be used if only values are needed that can be generated on the spot.
```